presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h

```diff
@@ -88,29 +103,36 @@ class LocalPersistentShuffleWriter : public ShuffleWriter {
   std::vector<size_t> inProgressSizes_;
   // The top directory of the shuffle files and its file system.
   std::string rootPath_;
+  std::string queryId_;
+  uint32_t shuffleId_;
   std::shared_ptr<velox::filesystems::FileSystem> fileSystem_;
   // Used to make sure files created by this thread have unique names.
   std::thread::id threadId_;
+  std::string nextAvailablePartitionFileName(
```

Comment:
Let's move this method before member variables and document it. Make sure to add empty lines before and after.